src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -237,6 +268,7 @@ public Builder indent(String indent) {
     }
 
     public JavaFile build() {
+      Collections.sort(staticImports);
```

Comment:
(I think my reluctance about this is that `build()` is mutating. A crazy person might call `build()` from two threads in parallel, and crash.)